presto-tests/src/test/java/com/facebook/presto/tests/TestTpchDistributedStats.java

```diff
@@ -26,9 +29,110 @@ public TestTpchDistributedStats()
     {}
 
     @Test
-    void testEstimateForSimpleQuery()
+    void testTableScanStats()
     {
-        assertEquals(compareMetric("SELECT * FROM NATION").result(), MATCH);
+        assertEquals(compareMetric("SELECT * FROM nation").result(), MATCH);
+        assertEquals(compareMetric("SELECT * FROM part").result(), MATCH);
+        assertEquals(compareMetric("SELECT * FROM partsupp").result(), MATCH);
+        assertEquals(compareMetric("SELECT * FROM region").result(), MATCH);
+        assertEquals(compareMetric("SELECT * FROM supplier").result(), MATCH);
+        assertEquals(compareMetric("SELECT * FROM lineitem").result(), MATCH);
+        assertEquals(compareMetric("SELECT * FROM lineitem").result(), MATCH);
     }
 
+    @Test
+    void testFilter()
+    {
+        String query = "" +
+                "SELECT * " +
+                "FROM lineitem " +
+                "WHERE l_shipdate <= DATE '1998-12-01' - INTERVAL '90' DAY";
+        MetricComparison metricComparison = compareMetric(query);
+        assertEquals(metricComparison.result(), DIFFER);
+        assertEquals(metricComparison.getEstimate().get(), 30000, 1);
```

Comment:
Then, the assert here would be:

`assertEquals(metricComparison.evaluate(equalTo(30000).withAbsoluteTolerance(1), MATCH)`

or, even better

`assertMatch(metricComparison.evaluate(equalTo(30000).withAbsoluteTolerance(1))`